src/platform/packages/private/kbn-ui-shared-deps-src/webpack.config.js

```diff
@@ -111,10 +113,19 @@ module.exports = {
     mainFields: ['browser', 'module', 'main'],
     conditionNames: ['browser', 'module', 'import', 'require', 'default'],
     alias: {
-      '@elastic/eui$': '@elastic/eui/optimize/es',
-      '@elastic/eui/lib/components/provider/nested$':
-        '@elastic/eui/optimize/es/components/provider/nested',
-      '@elastic/eui/lib/services/theme/warning$': '@elastic/eui/optimize/es/services/theme/warning',
+      // @elastic/eui-amsterdam is a package alias defined in Kibana's package.json
+      // that points to special EUI releases bundled with Amsterdam set as the default theme
+      // and meant to be used with Kibana 8.x. Kibana 9.0 and later use the Borealis theme
+      // and should use the regular @elastic/eui package.
```

Comment:
@tkajtoch can we setup a TODO to remind us to remove this integration once the old theme is no longer used? :D

Reply:
Added a todo and an issue [here](https://github.com/elastic/kibana/pull/219818/commits/61c6e2e06bc57cb668c4b022f5e53aad81596aac) :D